reana-ui/src/pages/signin/Signin.js

```diff
@@ -59,9 +59,28 @@ export default function Signin() {
       <Segment>
         {config.cernSSO && (
           <>
-            <Button basic fluid size="large" onClick={handleClick}>
+            <Button
+              basic
+              fluid
+              size="large"
+              onClick={() => handleClick("cern_openid")}
+            >
               Sign in with CERN Single Sign-On
             </Button>
+            {(config.localUsers || config.loginEnabled) && <Divider></Divider>}
+          </>
+        )}
```

Comment:
I removed it and added a small margin instead as otherwise, the buttons are directly beneath each other, which does not look very appealing. Also changed the logic to display the local user OR divider if either button is configured.